src/points/saga.ts

```diff
@@ -275,6 +276,15 @@ function* watchTrackPointsEvent() {
 
 export function* watchAppMounted() {
   yield* take(AppActions.APP_MOUNTED)
+  yield* call(initializePoints)
+}
+
+export function* watchInitializeAccountSuccess() {
+  yield* take(AccountActions.INITIALIZE_ACCOUNT_SUCCESS)
+  yield* call(initializePoints)
+}
+
+function* initializePoints() {
```

Comment:
This works, note we could also do this (though `getWalletAddress` should probably be updated to reflect what it waits).

```suggestion
  // This waits for the wallet address to exist before proceeding
  yield* call(getWalletAddress)
```

I guess since `initializePoints` `spawns` sagas, it's unclear without looking whether they are long running or not, and whether it's "safe" to spawn multiple instances so maybe preferable to do it only once and wait for the account?

Reply:
@jeanregisser oh, thanks for this suggestion! it makes the change a lot simpler.